zktest/test/java/org/zkoss/zktest/zats/test2/B85_ZK_3972Test2.java

```diff
@@ -0,0 +1,28 @@
+/* B85_ZK_3972Test2.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Mon Jun 25 15:24:06 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/
+package org.zkoss.zktest.zats.test2;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.zkoss.zktest.zats.ZATSTestCase;
+
+public class B85_ZK_3972Test2 extends ZATSTestCase {
```

Comment:
You can rename to `B85_ZK_3972_2Test`, and it will find `B85-ZK-3972-2.zul`.

All test class name should be ended with Test.

Reply:
ok